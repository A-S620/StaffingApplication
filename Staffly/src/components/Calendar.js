import React, {Component} from 'react';


class Calendar extends Component {
    render() {
        return (
            <div className="Calendar" style={{marginTop: 60}}>
                <div>
                    test 2
                </div>
            </div>
        );
    }
}

export default Calendar;