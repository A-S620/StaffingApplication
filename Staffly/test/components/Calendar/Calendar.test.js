import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux'
import Calendar from "../../../src/components/Calendar";
describe('App ', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Calendar debug />);

        expect(component).toMatchSnapshot();
    });
});