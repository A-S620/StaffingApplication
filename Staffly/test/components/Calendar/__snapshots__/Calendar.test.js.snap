// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  should render correctly in "debug" mode 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  className="makeStyles-root-1"
  direction="column"
  justify="flex-start"
  style={
    Object {
      "marginTop": 30,
    }
  }
>
  <LeaveCalendar />
</WithStyles(ForwardRef(Grid))>
`;
